src/components/BarChart/BarChart.jsx

```diff
@@ -256,6 +274,23 @@ const BarChart = createClass({
 		};
 	},
 
+	components: {
+		/**
+		 * Body content for the message to display when the chart has no data.
+		 */
+		EmptyMessageBody: createClass({
```

Comment:
I think these could just be references to `EmptyStateWrapper.Body` and `EmptyStateWrapper.Title`